lib/libYulAST/IntrinsicPatterns.cpp

```diff
@@ -198,4 +198,36 @@ IntrinsicPatternMatcher::parseYulStructType(std::string_view name) {
   return res;
 }
 
+ArrayTypeResult IntrinsicPatternMatcher::parseArrayType(std::string_view name) {
+  ArrayTypeResult res;
```

Comment:
nit: let's initialize this just in case
```suggestion
  ArrayTypeResult res{};
```

Reply:
done